Octokit/Models/Request/SearchIssuesRequest.cs

```diff
@@ -337,4 +336,74 @@ public enum IssueTypeQualifier
         [Parameter(Value = "issue")]
         Issue
     }
+
+    public class RepositoryCollection : IEnumerable<string>
```

Comment:
:thumbsup: to more specialized collection

Not sure if the PCL library might get in the way of using a concrete class here though...